.gitmodules

```diff
@@ -39,3 +39,7 @@
 	path = repos/xgcm/xgcm-examples
 	url = https://github.com/xgcm/xgcm-examples.git
 	branch = binderbot-built
+[submodule "repos/pangeo-gallery/ncar-gallery"]
+	path = repos/pangeo-gallery/ncar-gallery
```

Comment:
This is wrong. It shoudl have been `repos/NCAR/ncar-gallery`. It needs to match the github repo https://github.com/NCAR/notebook-gallery. I'll fix it.